windows_bat/PokemonGo-Bot-Install.bat

```diff
@@ -203,14 +201,9 @@ if exist %PGBotPath%\encrypt. copy %PGBotPath%\encrypt. %DownPath%
 if exist %PGBotPath% rmdir %PGBotPath% /s /q
 if not exist %PGBotPath% md %PGBotPath%
 cd C:\Python27\
-pip2 install --upgrade virtualenv

```

Comment:
Should this not be presented as an option to the user whether they want virtualenv or not?